nautobot/extras/tests/test_views.py

```diff
@@ -1362,7 +1362,7 @@ class JobTestCase(
 
     def _get_queryset(self):
         """Don't include hidden or non-installed Jobs as they won't appear in the UI by default."""
-        return self.model.objects.filter(installed=True, hidden=False)
+        return self.model.objects.filter(installed=True, hidden=False, has_sensitive_variables=False)
```

Comment:
Instead of filtering these jobs out we should add the required fields to the form data in `setUpTestData`

```suggestion
        return self.model.objects.filter(installed=True, hidden=False)
```

On line 1423:

```py
            "has_sensitive_variables": False,
            "has_sensitive_variables_override": True,
```

Reply:
Good thought.